source/auth/auth.rst

```diff
@@ -254,6 +254,17 @@ or "SCRAM-SHA-256").
 The cache entry value MUST be either the ``saltedPassword`` parameter or the
 combination of the ``clientKey`` and ``serverKey`` parameters.
 
+Reauthentication
+~~~~~~~~~~~~~~~~
+
+On any operation that requires authentication, the server may raise the
+error ``ReauthenticationRequired`` (391) if the user's credential has expired.
+Drivers MUST immediately attempt to force a reauthenication on the connection
```

Comment:
Should we also be more specific here on what reauthentication exactly entails? Is this the step1/step2 flow (assuming we remove the cached credentials potentially) or is it the full handshake steps with speculative auth? And will we have more prose tests added to test this? The current unified tests are only testing retryability but don't have anything to do with auth specifically.

Reply:
I updated the language.  I'm not sure of a way to test it that will not result in a race condition, because we need the initial token to succeed, but be "about to expire".

Reply:
Thanks for the language update and understand the test issue. I'll mark this comment resolved - great work so far!